### PR TITLE
[OS-909] - Add 3 Dashboard icons to open Jupyter Notebooks

### PR DESCRIPTION
**Pull request by skarbat**
This PR adds 3 icons on the dashboard launch pad to open:

 * 2 Kano Jupyter sample notebooks
 * 1 Root view of all samples provided by the package

Package built and tested on the PI, works fine.

### CODE REVIEW

File: config/dashboard/kano-jupyter-strings.app [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "title": "Jupyter: Strings",
+    "tagline": "Jupyter Notebook Frontend",
+    "description": "Are you ready to interact with Python on a Browser? Hit Kano Jupyter and enjoy live coding!",
+    "slug": "jupyter",
+    "icon": "kano-jupyter-strings",
+    "colour": "#4A5152",
+    "categories": ["code"],
+    "packages": [],
+    "dependencies": ["kano-jupyter", "kano-profile"],
+    "launch_command": "kano-tracker-ctl session run 'kano-jupyter' 'kano-jupyter run --notebook=strings/Fun.ipynb'",
```
Comment (tombettany) [inline config/dashboard/kano-jupyter-strings.app]: Might want to distinguish the different notebooks from a tracking perspective. Maybe we shouldn't worry about it now until we have final versions of each of these.

File: config/dashboard/kano-jupyter-strings.app [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "priority": 0
```
Comment (tombettany) [inline config/dashboard/kano-jupyter-strings.app]: Likewise with `priority`, we may want to set it but probably won't worry right now.

**Review by tombettany — APPROVED** [review]
Excellent stuff. Only very minor comments about things that we should do in the future when we have fully decided the exact content.

File: config/dashboard/kano-jupyter-strings.app (hunk 1 above)
Comment (skarbat) [reply config/dashboard/kano-jupyter-strings.app]: Great idea, fixed that to distinguish tracking

File: config/dashboard/kano-jupyter-strings.app (hunk 2 above)
Comment (skarbat) [reply config/dashboard/kano-jupyter-strings.app]: For the sake of simplicity I've done that too - attached screenshot to Jira ticket. Thanks!

**Pull request merged**